packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -763,16 +763,36 @@ abstract class AndroidViewController extends PlatformViewController {
   /// Sends the message to dispose the platform view.
   Future<void> _sendDisposeMessage();
 
+  /// True if [_sendCreateMessage] can only be called with a non-null size.
+  bool get _createRequiresSize;
+
   /// Sends the message to create the platform view with an initial [size].
+  ///
+  /// If [_createRequiresSize] is true, it is a programming error to call this
+  /// with a null size, and the call should instead be deferred until the size
+  /// is available.
   Future<void> _sendCreateMessage({Size? size});
```

Comment:
Should we make size covariant and nullable on a subclass?

Reply:
Nice, it hadn't occurred to me that `covariant` would work for nullability. Simplified by making it covariant, and non-nullable for the subclass where it's required.